index.d.ts

```diff
@@ -44,7 +44,7 @@ export interface AxiosRequestConfig {
   baseURL?: string;
   transformRequest?: AxiosTransformer | AxiosTransformer[];
   transformResponse?: AxiosTransformer | AxiosTransformer[];
-  headers?: any;
+  headers?: object | undefined;
```

Comment:
`headers?` includes `undefined`. And `object` in Typescript is a little wired, which means everything except primitive types, i.e. "let a: object = [];" is valid.

Personally, I suggest,

```
headers?: {[key: string]: string}
```